### DISCUSSION TITLE
I am a high school student. please help me

### DISCUSSION THREAD

**Discussion by yesmeyunjin** [community/community#132954] [Programming Help]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

I tried to run autodock for 15 hours, but I ended up failing. I made all the pdb files for target and ligand, and even made a pdbqt file through openbabel, but it keeps getting errors. If I put a pdbqt file in it, it doesn't even work, so I don't know what to do. This is a pdb file of antibody K that I made myself. I used alphafold2 colab. Why doesn't it work? It says, "The program Prepare_Ligand 4 has not finished."

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @yesmeyunjin ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
